### DISCUSSION TITLE
Add a setoption to disable default button1 behaviour

### DISCUSSION THREAD

**Discussion by MadDoct** [arendst/Tasmota#10216] [Q&A] [answered]
Would it be possible to add a setoption to disable the default behaviour of button1 (X number of presses activate relayX) and make it behave like any other button (single press activates relay1 and other presses only report state changes)?
When you have more than a button, there's no advantage in havind the default behaviour and the way it is now, it limits the usage of multipress on that button...

**Comment by vic42** [comment, marked as answer]
What if you set `SetOption73 1` and do the button processing in a rule?

**Comment by MadDoct** [comment]
I know that's an option, but will require a rule for each button instead of "just" disabling a specific feature... If possible, it would be ideal, IMHO... If not, rules it is... ;)
